### PR TITLE
Allowing Wizard to navigate if children exist but their view isn't initialised

### PR DESCRIPTION
**Pull request by TalmizAhmed**
This fix was needed to allow navigation inside wizard in comparing two forms
https://jira.corp.adobe.com/browse/FORMS-10513
## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js [hunk 1]
```diff
@@ -290,7 +290,14 @@
             var activeTabElement = this.#getCachedTabs()[activeIndex];
             var activeChildId = activeTabElement.id.substring(0, activeTabElement.id.lastIndexOf(Wizard.#tabIdSuffix));
             var activeChildView = this.getChild(activeChildId);
-            var validationErrorList = activeChildView.getModel().validate();
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:293]: Why are we still using var in this code ?

Comment (rismehta) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:293]: Please add test case for the use-case

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js (hunk 1 above)
Comment (TalmizAhmed) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js:293]: yeah I'm not sure why the whole file doesn't use any ES6. As for test case, the use case is for Compare forms from formsmanager, so will add it wherever we add tests for versioning, copare etc.

**Pull request merged**